MibTeX/src/de/mibtex/export/typo3/Typo3Entry.java

```diff
@@ -0,0 +1,185 @@
+package de.mibtex.export.typo3;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.jbibtex.BibTeXEntry;
+
+import de.mibtex.BibtexEntry;
+import de.mibtex.export.ExportTypo3Bibtex;
+
+public class Typo3Entry implements Comparable<Typo3Entry> {
+	private static Map<String, String> ToURLOverwrites = new HashMap<>();
+	static {
+		ToURLOverwrites.put("&auml;", "�");
+		ToURLOverwrites.put("&ouml;", "�");
+		ToURLOverwrites.put("&uuml;", "�");
+		ToURLOverwrites.put("&Auml;", "�");
+		ToURLOverwrites.put("&Ouml;", "�");
+		ToURLOverwrites.put("&Uuml;", "�");
+		ToURLOverwrites.put("&8211;", "-");
+		ToURLOverwrites.put("?", "?");
+		ToURLOverwrites.put(":", ":");
+		ToURLOverwrites.put("/", "/");
+		ToURLOverwrites.put("#", "#");
+		ToURLOverwrites.put("&szlig;", "�");
+	}
+	
+	public final BibtexEntry source;
+	
+	public String type; // inproceedings, article, ...
+	public String key;
+	
+	public String typeAttrib; // type = {...}
+	public String title;
+	public List<String> authors;
+	public List<String> editors;
+	public String booktitle;
+	public String address;
+	public String publisher;
+	public String journal;
+	public String location;
+	public String school;
+	public String pages;
+	public int year;
+	
+	public String doi;
+	public String isbn;
+	public String issn;
+	
+	public String note;
+	
+	public List<String> tags;
+	
+	public Typo3Entry(BibtexEntry bib, Map<String, String> variables) {
```

Comment:
Extract methods from constructor

Reply:
Do you want me to create an initializeX method for each field X similar to BibtexEntry.java?
If so, why?

Reply:
This constructor has more than 60 lines and there is a rule of thumb: if it has more than 7 lines, rethink whether you can split the method/constructor.

Reply:
I agree in general. The problem here is that Typo3Entry has 19 attributes, each of which should be initialized in the constructor. Initialization is a one-liner for each of them now (after the last commits in #38) except for authors for which extraction to a seperate method does not seem reasonable to me.